city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,97 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa69Spider(CityScrapersSpider):
+    name = "chi_ssa_69"
+    agency = "Chicago Special Service Area #69 95th & Ashland Avenue"
+    timezone = "America/Chicago"
+
+    @property
+    def start_urls(self):
+        today = datetime.now()
+        last_week = today - timedelta(days=7)
+        in_two_months = today + timedelta(days=60)
+        chicago_timezone = "America/Chicago"
+
+        return [(
+            "https://www.googleapis.com/calendar/v3/calendars/gagdcchicago%40gmail.com/events"
+            "?calendarId=gagdcchicago@gmail.com&singleEvents=true&timeZone={}&"
+            "sanitizeHtml=true&timeMin={}T00:00:00{}&timeMax={}T00:00:00{}&"
+            "key=AIzaSyC-KzxSLmmZitsCVv2DeueeUxoVwP0raVk"
+        ).format(
+            chicago_timezone, last_week.strftime("%Y-%m-%d"), chicago_timezone,
```

Comment:
I'm not sure that including `America/Chicago` where `-05:00` used to be works the same way, and it looks like it's returning an error code in the CI output https://github.com/City-Bureau/city-scrapers/runs/276393906#step:9:71. I think it's fine to just leave a static offset of 5 hours in the locations next to the datetimes since we'll only be off by an hour max

Reply:
gotcha

Reply:
Sorry, I think I wasn't clear here. `timeZone` still needs to be `America/Chicago` so that it will automatically adjust the offset, but the offset calculation should be included separately so that it doesn't trigger a server error. I think it's fine to just include the `-05:00` statically in the string:

```python
        [(
            "https://www.googleapis.com/calendar/v3/calendars/gagdcchicago%40gmail.com/events"
            "?calendarId=gagdcchicago@gmail.com&singleEvents=true&timeZone={}&"
            "sanitizeHtml=true&timeMin={}T00:00:00-05:00&timeMax={}T00:00:00-05:00&"
            "key=AIzaSyC-KzxSLmmZitsCVv2DeueeUxoVwP0raVk"
        ).format(
            chicago_timezone,
            last_week.strftime("%Y-%m-%d"),
            in_two_months.strftime("%Y-%m-%d")
        )]
```